Gradle.md

```diff
@@ -161,6 +161,46 @@ The summary looks typically like in the screenshot below.
 
 ![image](https://user-images.githubusercontent.com/231070/146644509-7b071436-3d04-46ee-a327-ea16ecfe9c76.png)
 
+### Performance characteristics
+
+<details>
+  <summary>Click to see more details about each of the calculated performance characteristics.</summary>
+
+#### Initial build time
+
+The elapsed time of the first build without any build performance acceleration measures explicitly applied by the experiment.
+
+#### Build time with instant savings
+
+The elapsed time of the second build with build performance acceleration measures explicitly applied by the experiment. The experienced build performance acceleration typically comes from the incremental build feature (experiment 1) or build caching (experiments 2 to 5).
+
+Achieving the build time with instant savings does not require any changes to the tasks of the build.
+
+#### Build time with pending savings
```

Comment:
We should consistently add a newline after each heading.

Reply:
Fixed in https://github.com/gradle/gradle-enterprise-build-validation-scripts/commit/938475e1a6cec5b541e755764f66b805e056b3af